go-controller/pkg/ovn/egressip_local_test.go

```diff
@@ -25,17 +25,13 @@ var _ = Describe("Local gateway mode EgressIP Operations with", func() {
 
 	getEgressIPStatusLenSafely := func(egressIPName string) func() int {
 		return func() int {
-			fakeOvn.controller.eIPAllocatorMutex.Lock()
-			defer fakeOvn.controller.eIPAllocatorMutex.Unlock()
```

Comment:
So do we need these `Safely` functions any more? (At least, the "Safely" is no longer meaningful is it?)

Reply:
You're right, I was going to remove "Safely" yesterday, but forgot. The functions are handy though, given how prevalent tests are using this.